Sources/Plasma/CoreLib/hsWindows.h

```diff
@@ -84,6 +84,11 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #       include <vld.h>
 #   endif // USE_VLD
 
+    // Conflicts with plDynamicTextMap
+#   ifdef DrawText
+#       undef DrawText
+#   endif
```

Comment:
It looks like you explicitly changed it to use `DrawTextA` though -- is this still needed?

Reply:
Yes, without this, `pyDynamicText::DrawText()` is impacted... But I could rename that method if you prefer.

Reply:
Ah, I missed what was conflicting here...  Nevermind then